src/main/java/fr/silverpricing/api/model/Coordinates.java

```diff
@@ -1,32 +1,59 @@
 package fr.silverpricing.api.model;
```

Comment:
:information_source: Package name does not match the directory structure. (Severity: Minor)